_wiki/FAQ/Windows.md

```diff
@@ -5,186 +5,157 @@ hide_from_auto_list: true
 
-Please make sure that you have no other tablet drivers on the system.\\
-To ensure your system is clean, follow the instructions for our [Tablet Driver Cleanup Utility](https://github.com/X9VoiD/TabletDriverCleanup/releases/latest), then resume here.
+- [Create a tablet support request in Github.](https://github.com/OpenTabletDriver/OpenTabletDriver/issues/new?assignees=&labels=configuration&projects=&template=tablet_configuration.yml&title=Add+support+for+)
+- [Create a tablet support thread in #config-creation channel of the Discord server.]({{ site.discord_invite_url }})
+- [Make your own tablet configuration.]({% link _wiki/Development/AddingTabletSupport.md %})
 
-Please check if your tablet requires WinUSB [here]({% link _sections/Tablets.md %}) (it will say in the notes).\\
-If it does, follow the [instructions]({% link _wiki/Install/Windows.md %}#winusb) and return here. Continue below for troubleshooting instructions if your tablet still isn't detected.
+#### Driver Cleanup
 
-If you are running OpenTabletDriver.UX, navigate from the top to the Console tab, if you are running the daemon standalone, look for a log that resembles the following:
+If your tablet is connected properly and is supported but is still not detected, please make sure that you have no other tablet drivers on the system.
 
-##### Tablet driver interefence
+To ensure your system is clean, follow the instructions in [TabletDriverCleanup](https://github.com/X9VoiD/TabletDriverCleanup/releases/latest), then resume here.
+
+Please check if your tablet requires WinUSB [here]({% link _sections/Tablets.md %}) (it will say in the Notes column). If it does, follow the [instructions]({% link _wiki/Install/Windows.md %}#winusb) and return here. Continue below for troubleshooting instructions if your tablet still isn't detected.
+
+#### Troubleshooting
+
+[Check logs]({% link _wiki/Documentation/Logs.md %}) for any errors or warnings. If you find any, try finding for a match and its accompanying solution below:
+
+##### Tablet driver interference
+
+_Symptoms_
 
-```sh
-Another tablet driver found: <name>
 ```
+Another tablet driver found: <driver>
+```
+
+_Solution_
 
-If you see this log message you should follow the instructions for [TabletDriverCleanup](https://github.com/X9VoiD/TabletDriverCleanup/releases/latest) if you haven't already, making sure to say yes to everything.\\
-If the log entry is still there after following the instructions and restarting OTD, try joining our support [Discord]({{ site.discord_invite_url }}) for further help.
+Follow the instructions in [TabletDriverCleanup](https://github.com/X9VoiD/TabletDriverCleanup/releases/latest), and make sure to type `y` for every driver found by the utility. Then, restart OTD. If the log entry is still present, try joining our support [Discord]({{ site.discord_invite_url }}) and asking for help in one of the #support channels.
```

Comment:
```suggestion
Follow the instructions in [TabletDriverCleanup](https://github.com/X9VoiD/TabletDriverCleanup/releases/latest), and make sure to type `y` for every driver found by the utility.

Then, restart OTD.

If the log entry is still present, try joining our support [Discord]({{ site.discord_invite_url }}) and asking for help in one of the #support channels.
```

or similar better spacing. I think it's easy to miss the restart step otherwise, though it can be done in 2 lines instead of 3 if necessary.